src/include/nodes/parsenodes.h

```diff
@@ -189,6 +189,12 @@ typedef struct Query
 	 * and executor needs this info to specifiy correct target also.
 	 * */
 	IntoClause *intoClause;
+
+	/*
+	 * GPDB_92_MERGE_FIXME: Used by CTAS to indicate if this query is part of
+	 * CTAS statement.
+	 */
+	bool		isCtas;
```

Comment:
isCtas is not needed here since there is intoClause. Unless we really do dispatch CreateTableAsStmt, but that could be modified in another patch. In that case, intoclause is in CreateTableAsStmt so we do not keep another copy of intoClause here (delivered to plan from Query) for dispatcher and we could use a bool variable here of course.

Reply:
If we retrieve the plan from plancache and do re-plan, there is no intoClause for query. That's why `isCtas` is needed here.